src/Chapter1.hs

```diff
@@ -632,7 +648,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+        let n' = abs n
+            lasttwo = n' `mod` 100
+            last = n' `mod` 10
+            penultimate = (lasttwo - last) `div` 10 in
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂

Reply:
Ah, yes, you're right! 🙈